devel-tools/update-changes

```diff
@@ -246,7 +246,7 @@ function add_to_changes_entry
         return 1;
     fi
 
-   if echo $msg | grep -q '^\(.*: *\)\{0,1\}Merge remote-tracking branch'; then # allow GH-XXX prefix
```

Comment:
Seems okay to check `$rev` for is merge via `git cat-file -p $rev | grep -c '^parent` or some such for more than one parent. `$git_author_email` and `$git_autor` do something similar.

Reply:
Nice, I adapted the patch to use that approach, PTAL.